ionic/src/app/set-wifi-password/set-wifi-password.component.ts

```diff
@@ -54,6 +56,10 @@ export class SetWifiPasswordComponent implements OnInit {
       this.errorMessage = err['error'];
     }
   }
-
+  async keepCheckingDeviceOnline() {
```

Comment:
we cannot make duplicate function. this function is already define somewhere else. use service for common function, but never duplicate

Reply:
this function write in common service thats why i remove form here